wireless/bluetooth/bt_hcicore.c

```diff
@@ -1741,15 +1741,6 @@ int bt_hci_cmd_send(uint16_t opcode, FAR struct bt_buf_s *buf)
           return -ENOBUFS;
         }
     }
-  else
-    {
-      /* We manage the refcount the same for supplied and created
-       * buffers so increment the supplied count so we can manage
-       * it as-if we created it.
-       */
-
-      bt_buf_addref(buf);
-    }
 
```

Comment:
The explanation in comment would be correct, if the function is a `neutral` function. When we pass a created buffer to `neutral` function, we also responsible to release the buffer in calling code.  But accordingly to its usage, this is a 'taking ownership' function.  The buffer comes with positive `ref` and here we are responsible to `release` it after usage.  No additional `addref` is needed.